lib/bokken/accounts/mentor.ex

```diff
@@ -42,9 +42,16 @@ defmodule Bokken.Accounts.Mentor do
   def changeset(mentor, attrs) do
     mentor
     |> cast(attrs, @required_fields ++ @optional_fields)
-    |> cast_embed(:socials, with: &Social.changeset/2)
     |> cast_attachments(attrs, [:photo], allow_urls: true)
     |> validate_required(@required_fields)
+    |> then(fn changeset ->
+      if changeset.params["socials"] == "[]" do
+        Map.update!(changeset, :params, &Map.put(&1, "socials", []))
+      else
+        changeset
+      end
+    end)
```

Comment:
```suggestion
    |> then(fn changeset ->
      if changeset.params["socials"] == "[]" do
        Map.update!(changeset, :params, &Map.put(&1, "socials", []))
      else
        changeset
      end
    end)
```
Put this in an auxiliary function instead of an anonymous one. Also, could you add other socials we discussed? Scratch, Trello, Discord and Slack?

Reply:
Done! 💪🏼